lib/config.ts

```diff
@@ -1,5 +1,5 @@
 let GRAPHQL_HOSTNAME = process.env.NEXT_PUBLIC_GRAPHQL_HOSTNAME as string
-const WEBSOCKET_HOSTNAME = process.env.NEXT_PUBLIC_WEBSOCKET_HOSTNAME as string
+const WEBSOCKET_URL = process.env.NEXT_PUBLIC_WEBSOCKET_URL as string
```

Comment:
don't cast

if `NEXT_PUBLIC_WEBSOCKET_URL` is not provided as env and someone do something like `WEBSOCKET_URL.length`, then it will crash the pod.